### ISSUE TITLE
gallbladder polyp

### ISSUE THREAD

**Issue by nklsbckmnn** [obophenotype/human-phenotype-ontology#11064]
**Preferred term label:**
 gallbladder polyp

**Synonyms**

**Definition (free text, please give PubMed ID)**
A polyp arising from the gallbladder mucosal wall.

**Parent term (use hpo.jax.org/app)**
Gallbladder mass (or Abnormal gallbladder morphology)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0009-0009-6791-365X

**Comment by pnrobinson** [comment]
@nklsbckmnn thanks!
Adding new term: 
- ID: HP:6001260
- label: Gallbladder polyp
- parents: [Abnormal gallbladder morphology (HP:0012437)](https://hpo.jax.org/app/browse/term/HP:0012437)
- pmid: PMID:38411693
- def: A gallbladder (GB) polyp is a growth that projects from the lining of the inside wall of the gallbladder into the lumen of the GB. GB polyps are a common incidental sonographic finding, the vast majority of which are benign.
- comment: Most benign GB polyps are non-neoplastic, including cholesterol pseudopolyps and inflammatory polyps.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11064
- synonyms:
